spring-integration-hazelcast/src/main/java/org/springframework/integration/hazelcast/message/HazelcastHeaders.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.integration.hazelcast.message;
+
+/**
+ * Hazelcast Message Headers
+ *
+ * @author Eren Avsarogullari
+ * @since 1.0.0
+ */
+public abstract class HazelcastHeaders {
+
+	private static final String PREFIX = "hazelcast_";
+
+	public static final String EVENT = PREFIX + "event";
```

Comment:
Ah! No, I see you just populate it as is - `EntryEventType`. 
I'd prefer to avoid non-java core types for `MessageHeaders` at all.
So, use for this header `getEventType().name()`, please.